portfolio/src/main/webapp/script.js

```diff
@@ -73,11 +73,11 @@ function contact() {
 
```

Comment:
This file doesn't look like it's changing in this PR

Reply:
? hm I don't think it should be moving for this particular change